code/modules/vehicles/wheelchair.dm

```diff
@@ -11,6 +11,7 @@
 	density = FALSE		//Thought I couldn't fix this one easily, phew
 	// Run speed delay is multiplied with this for vehicle move delay.
 	var/delay_multiplier = 6.7
+	var/overlay_icon = "wheelchair_overlay"
```

Comment:
I see this variable defined and re-defined, but not used. Is it WIP?
Document it, as well.

Reply:
Oh I thought I used it, I must've forgotten to, thanks.

Reply:
Could you document it?